tests/window_finder/test_sorted_by_least_number_of_satellites.py

```diff
@@ -1,36 +1,162 @@
 from datetime import datetime, timedelta
 from typing import List
-
+from pathlib import Path
+import pytz
+from tests.utilities import get_script_directory
 from satellite_determination.custom_dataclasses.frequency_range import FrequencyRange
 from satellite_determination.custom_dataclasses.overhead_window import OverheadWindow
 from satellite_determination.custom_dataclasses.reservation import Reservation
 from satellite_determination.custom_dataclasses.satellite.satellite import Satellite
 from satellite_determination.custom_dataclasses.time_window import TimeWindow
+from satellite_determination.custom_dataclasses.satellite.tle_information import TleInformation
+from satellite_determination.custom_dataclasses.satellite.international_designator import InternationalDesignator
+from satellite_determination.custom_dataclasses.satellite.mean_motion import MeanMotion
 from satellite_determination.window_finder import SuggestedReservation, WindowFinder
+from satellite_determination.event_finder.event_finder_rhodesmill.event_finder_rhodesmill import EventFinderRhodesMill
 from tests.window_finder.definitions import ARBITRARY_FACILITY
-from tests.window_finder.support.validator_satellites_are_overhead_at_specific_times import \
-    ValidatorSatellitesAreOverheadAtSpecificTimes
+
 
 
 _ARBITRARY_FREQUENCY_RANGE = FrequencyRange(frequency=2., bandwidth=1.)
 
 
 class TestSortedByLeastNumberOfSatellites:
-    def test(self):
+
+    def test_search(self):
         suggestions = WindowFinder(
             ideal_reservation=self._ideal_reservation,
-            satellites=[window.satellite for window in self._overhead_windows],
-            validator=ValidatorSatellitesAreOverheadAtSpecificTimes(
-                overhead_times=[window.overhead_time for window in self._overhead_windows]),
-            start_time_increments=timedelta(days=1),
-            search_window=timedelta(weeks=1)
-        ).find()
-        assert suggestions == self._expected_suggestions
+            satellites=Satellite.from_tle_file(
+                tlefilepath=Path(get_script_directory(__file__), 'international_space_station_tle.tle'), \
+                frequencyfilepath=Path(get_script_directory(__file__), 'fake_ISS_frequency_file.csv')),
+            event_finder=EventFinderRhodesMill,
+            start_time_increments=timedelta(hours=4),
+            search_window=timedelta(days=1)
+        ).search()
+        #assert suggestions == self._expected_suggestions_search
+        for sug in suggestions:
+            print(sug.suggested_start_time)
+            print(len(sug.overhead_satellites))
+            for sat in sug.overhead_satellites:
+                print(sat.satellite.name)
```

Comment:
Printing instead of asserting?

```suggestion
        assert suggestions == self._expected_suggestions_search
```

Reply:
Fixed